app/services/induction/change_induction_record.rb

```diff
@@ -14,12 +19,24 @@ def call
         }
         new_record = induction_record.dup
 
-        if induction_record.end_date.present? && induction_record.end_date > time_now
-          default_attrs[:school_transfer] = induction_record.school_transfer?
-          induction_record.update!(school_transfer: false)
+        if induction_record.end_date.present?
+          if induction_record.end_date < time_now
+            # We're going to possibly be inserting a new record into an existing chain so
```

Comment:
I've had a long think about this, with three examples, one where the start date is before the end date. one where it is possible that the start date is after the end date, and one where they are the same.

The main issue I see here for all examples is that we are rewriting history, also where the start date is not the real date the change happened which I think is key, and end date on the previous induction record is also not the real date the change happened which is the current time, I find this will affect our timeline of when things happened and will really mess with our audits as well.

Since we are using the indication that the end date is present on an induction record, I'd like to explore options where we query those induction records, and happy to chat through any other options we might have @tonyheadford